teste p1 adicionado

andrewrosemberg opened the pull request:
teste p1 adicionado Andrew e Bianca

Commit:
teste p1 adicionado

Commit:
better expansion

Commit:
Merge branch 'master' into Branch-Andrew4

joaquimg commented on miptests.jl:
```diff
@@ -1152,6 +1152,124 @@ function test_rv_6(solveMIP::Function, solver::MathProgBase.AbstractMathProgSolv
+
+  #######################################################
+    @testset "Alocacao de portifolio Viavel" begin
+        myModel = Model(solver = solver)
```
Essa linha tem que ir pra fora do test set

joaquimg commented on miptests.jl:
```diff
@@ -1152,6 +1152,124 @@ function test_rv_6(solveMIP::Function, solver::MathProgBase.AbstractMathProgSolv
+        @test U_buy ≈ getValue(u_buy) atol=1E-07
+        @test U_sell ≈ getValue(u_sell) atol=1E-07      
+    end
+    setoutputs!(m,solution,testresult)
```
e aqui temos que coloca o nome certo

joaquimg reviewed (commented):
Tem duas pequenas modificações a fazer, e seria bomse vc ja adicionasse a linha correspondente no indice de testes que está definito no runmiptests.jl

Com esse lista é facinho rodar tudo

Commit:
fazendo correcoes

nome do modelo corrigido na chamada da função setoutputs, definição do modelo fora do test set e criação da variável  testresult

andrewrosemberg commented:
corrigido @joaquimg 
so falta eu adicionar la na sua funcao a linha para esse teste

Commit:
Adicionando problema da g1

adicionando linha com problema da g1

Commit:
Merge branch 'master' into Branch-Andrew4

andrewrosemberg commented:
resolved conflicts

joaquimg commented:
boa!
